puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/release/foreman/ga.groovy

```diff
@@ -0,0 +1,93 @@
+pipeline {
+    agent none
+
+    options {
+        timestamps()
+        timeout(time: 2, unit: 'HOURS')
+        disableConcurrentBuilds()
+        ansiColor('xterm')
+    }
+
+    stages {
+        stage('Mash Koji Repositories') {
+            agent { label 'sshkey' }
+            steps {
+                mash("foreman-mash-split-${major_version}.py")
+            }
+        }
+        stage('Repoclosure') {
+            agent { label 'el' }
+            steps {
+                git url: "https://github.com/theforeman/foreman-packaging", branch: "rpm/${major_version}", poll: false
+
+                def command = [
+                    "./repoclosure.sh yum_el7.conf",
+                    "http://koji.katello.org/releases/yum/foreman-${major_version}/RHEL/7/x86_64/",
+                    "-l el7-foreman-rails-${major_version}",
+                    "-l el7-base",
+                    "-l el7-updates",
+                    "-l el7-epel",
+                    "-l el7-extras",
+                    "-l el7-scl",
+                    "-l el7-puppet-5"
+                ]
+
+                dir('repoclosure') {
+                    sh command.join(" ")
+                }
+            }
+            post {
+                always {
+                    deleteDir()
+                }
+            }
+        }
+        stage('Install Test') {
+            steps {
+                def operatingsystems = ['centos7', 'debian9', 'ubuntu1604', 'ubuntu1804']
+                def pipelines = operatingsystems.collect { os ->
+                    [
+                        name: os,
+                        job: 'foreman-release-test',
```

Comment:
this will generate 4 jobs with the same name, which will probably fail

Reply:
also, there is no such job at the moment. should it be in this PR too?

Reply:
for Katello, we have a job per major version (technically, minor, 3.x, same lingo error as in the rest of our jobs :D)